pkg/scale/varying_data_type.go

```diff
@@ -95,7 +96,7 @@ func NewVaryingDataType(values ...VaryingDataTypeValue) (vdt VaryingDataType, er
 func MustNewVaryingDataType(values ...VaryingDataTypeValue) (vdt VaryingDataType) {
 	vdt, err := NewVaryingDataType(values...)
 	if err != nil {
-		panic(err)
+		panic(fmt.Sprintf("adding varying data type value: %s", err))
```

Comment:
```suggestion
		panic(err)
```

this doesn't need to be wrapped